suite-native/module-send/src/components/SendScreen.tsx

```diff
@@ -19,7 +19,6 @@ export const SendScreen = ({ children, footer, screenHeader, subheader }: SendSc
             screenHeader={screenHeader}
             subheader={subheader}
             footer={footer}
-            keyboardShouldPersistTaps="never"
```

Comment:
this probably means, that keyboardShouldPersistTaps is never set anywhere, maybe to remove the prop?

Reply:
removed d6fb6a99c8